token/cli/src/main.rs

```diff
@@ -2613,6 +2621,15 @@ fn app<'a, 'b>(
                             Rate authority defaults to the mint authority."
                         ),
                 )
+                .arg(
+                    Arg::with_name("metadata-address")
```

Comment:
Otherwise it won't ever get picked up!
```suggestion
                    Arg::with_name("metadata_address")
```